### PR TITLE
[config] Add Zeiss API connection test microscope file

### PR DESCRIPTION
**Pull request by pieleric**
Almost all SEM APIs have a corresponding test microsope file, which only
uses the SEM. The Zeiss didn't have one yet.

### CODE REVIEW

**Review by FCanberk — APPROVED** [review]

File: install/linux/usr/share/odemis/hwtest/zeiss-only.odm.yaml [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+"EBeam Focus": {
+    role: ebeam-focus,
+}
+
+"Sample Stage": {
+    role: stage,
+    affects: ["SEM E-beam full"],
+    init: {
+        inverted: ["x", "y"], # TODO: z ??
```
Comment (sneepstefan) [inline install/linux/usr/share/odemis/hwtest/zeiss-only.odm.yaml:123]: Does this todo still need to be in?

Comment (pieleric) [reply install/linux/usr/share/odemis/hwtest/zeiss-only.odm.yaml:123]: I guess with the METEOR work, we should now know whether the Z goes up or down. Normally the convention in Odemis is that an increase in Z causes the sample to be closer to the e-beam source. 
Do you know?

Comment (sneepstefan) [reply install/linux/usr/share/odemis/hwtest/zeiss-only.odm.yaml:123]: Good question we have to check with Jochem and Karisma for that.

Comment (sneepstefan) [reply install/linux/usr/share/odemis/hwtest/zeiss-only.odm.yaml:123]: z is positive upwards I just heard from Karishma so I guess we can leave inversion of z out.

Comment (sneepstefan) [inline install/linux/usr/share/odemis/hwtest/zeiss-only.odm.yaml:123]:
```suggestion
        inverted: ["x", "y"],
```

**Comment by pieleric** [comment]
@sneepstefan , I fixed the comment about Z, and many other comments. Could you give a second look?

File: install/linux/usr/share/odemis/hwtest/zeiss-only.odm.yaml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+# Connect:
+# X -> AO 0
+# Y -> AO 1
+# Ground -> AO GND
+"SEM E-beam": {
+    role: null,
+    init: {
+        channels: [0, 1],
+        # On Delmic scanning box v2, the voltage is x2, so need to specify twice smaller values than needed.
```
Comment (sneepstefan) [inline install/linux/usr/share/odemis/hwtest/zeiss-only.odm.yaml:57]: This comment does not seem to be belong to the next line?

File: install/linux/usr/share/odemis/hwtest/zeiss-only.odm.yaml [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+        # 1 = Open drain output (Y0.0)
+        # 2 = Digital Out 1
+        # 3 = Digital Out 0
+        # 4 = Status led
+        # output ports -> True (indicate scanning) or False (indicate parked)
+        scanning_ttl: {
+            3: [True, "external"],
+            4: True
+        },
+        settle_time: 120.e-6, # s
```
Comment (sneepstefan) [inline install/linux/usr/share/odemis/hwtest/zeiss-only.odm.yaml:72]: 0.12 ms sounds extremely low.. not sure if this is usual

File: install/linux/usr/share/odemis/hwtest/zeiss-only.odm.yaml (hunk 2 above)
Comment (pieleric) [reply install/linux/usr/share/odemis/hwtest/zeiss-only.odm.yaml:57]: Good point.

File: install/linux/usr/share/odemis/hwtest/zeiss-only.odm.yaml (hunk 3 above)
Comment (pieleric) [reply install/linux/usr/share/odemis/hwtest/zeiss-only.odm.yaml:72]: That's the time needed for the flyback after scanning a whole line. With short dwell time of 1µs, which is relatively common, that's equivalent to 120 pixels. So, no, it's not really small. Actually some SEM even supports settle times as small as 10µs.

**Comment by pieleric** [comment]
@sneepstefan , I've fixed the last comment. Do you mind to do one more review?

**Review by sneepstefan — APPROVED** [review]

**Pull request merged**